agentarchives/archivists_toolkit/client.py

```diff
@@ -1,5 +1,8 @@
 import logging
 
+import pymysql
+pymysql.install_as_MySQLdb()
+
 import MySQLdb
```

Comment:
Yes; `pymysql.install_as_MySQLdb()` doesn't create a module in the local namespace called `MySQLdb`, but instead allows `import MySQLdb` to load `pymysql`.